examples/Utils/referenceDataReq.js

```diff
@@ -0,0 +1,29 @@
+const uAPI = require('../../index');
+const config = require('../../test/testconfig');
+
+function uuidv4() {
```

Comment:
Probably https://www.npmjs.com/package/uuid in devDependencies won't harm

Reply:
Yea.. did not notice that was there